libs/langchain/langchain/chains/natbot/base.py

```diff
@@ -5,15 +5,27 @@
 import warnings
 from typing import Any, Dict, List, Optional
 
+from langchain_core._api import deprecated
 from langchain_core.callbacks import CallbackManagerForChainRun
 from langchain_core.language_models import BaseLanguageModel
+from langchain_core.output_parsers import StrOutputParser
 from langchain_core.pydantic_v1 import root_validator
+from langchain_core.runnables import Runnable
 
 from langchain.chains.base import Chain
-from langchain.chains.llm import LLMChain
 from langchain.chains.natbot.prompt import PROMPT
 
 
+@deprecated(
+    since="0.2.13",
+    message=(
+        "Importing NatBotChain from langchain is deprecated and will be removed in "
+        "langchain 1.0. Please import from langchain_community instead: "
+        "from langchain_community.chains.natbot import NatBotChain. "
+        "You may need to pip install -U langchain-community."
+    ),
+    removal="1.0",
+)
```

Comment:
should we do the thing where we just re-import this?

Reply:
i also dont think this needs to be a chain at all?

Reply:
I considered re-importing but because langchain no longer depends on langchain-community, this would be a breaking change. Can do if preferred.

IMO this should be a community integration, which is why I set up imports there. The [natbot](https://github.com/nat/natbot) repo has had no activity for 2 years and we originally specified this as "deprecate without replacement." So didn't think refactoring it is high priority.